### PR TITLE
(docs): kitty usage

### PR DESCRIPTION
**Pull request by windowsrefund**
Add instructions on how to use tinty with kitty terminal

### CODE REVIEW

File: USAGE.md [hunk 1]
```diff
@@ -502,6 +503,27 @@ However, limited testing has shown `dunst` will not complain if its configuratio
...
+```
+
+Add the following to `~/.config/tinted-theming/tinty/config.toml`:
+
+```toml
+[[items]]
+path = "https://github.com/tinted-theming/tinted-kitty"
+name = "tinted-kitty"
+themes-dir = "colors"
+hook = "cp -f %f ~/.config/kitty/current-theme.conf && for i in $(pgrep kitty); do kitten @ --to unix:/tmp/kitty-${i}.sock set-colors -a -c ~/.config/kitty/current-theme.conf; done"
```
Comment (phantomwhale) [inline USAGE.md:523]: I have the following in my tinty config:

`hook = "kitty @ set-colors -a -c %f"`

Which seems to work on my setup, and might be simpler?

Comment (phantomwhale) [reply USAGE.md:523]: (I also don't have the include directive in my kitty.conf file; remote control is enabled though, and listen_on is enabled to /tmp/kitty, but that's for a different plugin from memory)

Comment (windowsrefund) [reply USAGE.md:523]: Interesting. I'll test it out against some edge cases I can imagine and report back.

Comment (windowsrefund) [reply USAGE.md:523]: According to `kitty.conf(5)` the socket used for remote control connections does actually apply to all kitty instances. This is not what I was expecting which is why I based everything on each PID. Instead, the PID is appended onto the socket dynamically so we end up with `/tmp/kitty-<pid>` for each instance we run. Very nice.

However, changing a theme using that hook just ends up refreshing the terminal I ran the `tinty apply` command from. My other terminal just sits there unimpressed :)

Regardless, I'll muck around with some version of the original bits in order to get something working that makes use of the simplified unix sockets. Just to be clear though, you're not saying you're able to update multiple kitty consoles with the setup you described, right?

**Comment by windowsrefund** [comment]
After mucking around with kitty for some time, I'm no longer interested in dealing with it and will just stick with Alacritty

File: USAGE.md (hunk 1 above)
Comment (phantomwhale) [reply USAGE.md:523]: This updates all the windows and tabs in all of my kitty instances; I usually only have one OS window running, but tried it with a second, and that seems to work as well.

CleanShot 2024-11-11 at 09 20 33
